components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/model/AuthenticatedIdPData.java

```diff
@@ -18,18 +18,36 @@
 
 package org.wso2.carbon.identity.application.authentication.framework.model;
 
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.identity.application.authentication.framework.config.model.AuthenticatorConfig;
 
 import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
 
 public class AuthenticatedIdPData implements Serializable {
 
     private static final long serialVersionUID = 5576595024956777804L;
```

Comment:
Please change the serialVersionUID since we changed the attribute in this class.
Otherwise, some serialization exceptions will be printed after an update.

Reply:
Since we persist the serialized object, we have to retain the same serialVersionUID to process stored objects which are serialized using the previous class definition.

Reply:
+1